api/src/main/java/uk/gov/dvsa/mot/vehicle/api/DateFormatAdapter.java

```diff
@@ -2,11 +2,19 @@
 
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import java.util.TimeZone;
 
 import javax.xml.bind.annotation.adapters.XmlAdapter;
 
 public class DateFormatAdapter extends XmlAdapter<String, Date> {
-    private final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+
+    private final SimpleDateFormat dateFormat;
+
+    public DateFormatAdapter() {
+
+        dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
```

Comment:
simple date format requires explicit timezone setting, otherwise it operates in system timezone which can be different when executed locally. It is safe in that AWS Lambda has this timezone fixed to UTC anyway.